Makefile

```diff
@@ -307,12 +307,39 @@ $(BUILD_DIRECTORY)/$(APPLICATION_NAME_ELECTRON)_$(APPLICATION_VERSION_DEBIAN)_$(
 		--extraMetadata.packageType=deb \
 		$(DISABLE_UPDATES_ELECTRON_BUILDER_OPTIONS)
 
-$(BUILD_DIRECTORY)/$(APPLICATION_NAME_LOWERCASE)-$(APPLICATION_VERSION)-$(TARGET_ARCH_APPIMAGE).AppImage: \
-	| $(BUILD_DIRECTORY)
-	$(NPX) build --linux AppImage $(ELECTRON_BUILDER_OPTIONS) \
+ifeq ($(TARGET_ARCH),x64)
+ELECTRON_BUILDER_LINUX_UNPACKED_DIRECTORY = linux-unpacked
+else
+ELECTRON_BUILDER_LINUX_UNPACKED_DIRECTORY = linux-$(TARGET_ARCH_ELECTRON_BUILDER)-unpacked
+endif
+
+$(BUILD_DIRECTORY)/$(ELECTRON_BUILDER_LINUX_UNPACKED_DIRECTORY):
```

Comment:
As I've (probably) mentioned before, it's always better to try and avoid having directories as Makefile-targets, because they can easily mess things up.
Would an alternative be to have `$(BUILD_DIRECTORY)/$(ELECTRON_BUILDER_LINUX_UNPACKED_DIRECTORY)/$(APPLICATION_NAME_ELECTRON)` as the target of this rule? (The last rule for this target would then need to be `touch $@` so that the datestamp gets updated when the target gets built)